cypress/integration/core/mpi/busqueda.spec.js

```diff
@@ -0,0 +1,56 @@
+
+context('MPI-Busqueda Paciente', () => {
+    let token
+    before(() => {
+        cy.login('38906735', 'asd').then(t => {
+            token = t;
+        });
+        cy.viewport(1280, 720);
+    })
+
+    beforeEach(() => {
+        cy.goto('/apps/mpi/busqueda', token);
+    });
+
+
+    it('buscar paciente por documento y verificar que no existe', () => {
+        cy.server();
+        cy.route('GET', '**api/core/mpi/pacientes**').as('busquedaConDocumento');
+        cy.get('button[class="btn btn-default dropdown-toggle waves-effect"]').should('have.prop', 'disabled', true);
```

Comment:
Cree un comando nuevo para manear plex-dropdown.

```suggestion
        cy.plexDropdown('label="NUEVO PACIENTE"').should('have.prop', 'disabled', true);
```

PErmite un segundo paramentro que puede ser un entero o un string. Si se pasa ese segundo paramentro se selecciona una opción del dropdown. Ya sea por indice o por el string que contenga esa opción. Ejemplo:


```
 cy.plexDropdown('label="NUEVO PACIENTE"', 0);
 
 ...

 cy.plexDropdown('label="NUEVO PACIENTE"', 'BEBÉ');
```

Reply:
Cambios realizados